src/Models/CourseModel.php

```diff
@@ -46,29 +46,37 @@ public function addCourse(array $newCourse): string
             `name`,
             `notes`,
             `in_person`,
-            `remote`
+            `remote`,
+            `in_person_spaces`,
+            `remote_spaces`
             ) 
             VALUES (
             :startDate, 
             :endDate, 
             :name,
             :notes,
             :in_person,
-            :remote);");
+            :remote,
+            :in_person_spaces,
+            :remote_spaces);");
 
         $startDate = $newCourse['startDate'];
         $endDate = $newCourse['endDate'];
         $name = $newCourse['courseName'];
         $notes = $newCourse['notes'];
         $in_person = $newCourse['in_person'];
         $remote = $newCourse['remote'];
+        $in_person_spaces = $newCourse['in_person_spaces'];
+        $remote_spaces = $newCourse['remote_spaces'];
```

Comment:
these variables are totally pointless, just use the array with the correct assoc key in the bindParam calls and remove these